.github/workflows/build-python.yml

```diff
@@ -72,3 +73,17 @@ jobs:
           name: html_unit_test
           path: unittest_results/*.html
           retention-days: 1
+  deploy-github-packages:
```

Comment:
i would suggest to ranme this to something pypi specific
"publish to pypi"?

Reply:
renamed to `deploy-pypi-packages`